src/prognostic_equations/vertical_diffusion_boundary_layer.jl

```diff
@@ -85,15 +85,14 @@ function vertical_diffusion_boundary_layer_tendency!(
     FT = eltype(Y)
     interior_uₕ = Fields.level(Y.c.uₕ, 1)
     interior_coordinates = Fields.level(Fields.coordinate_field(Y.c), 1)
```

Comment:
```suggestion
```

Reply:
Ah, this is still used to get `ᶜΔz_surface`, but I'll grab it from `interior_uₕ` instead.

Reply:
Ah ok, thanks! I was on my phone and didn’t see the whole file 😂

Reply:
Actually, does Δz_field for a center field give the interval between cell center or cell face?

Reply:
> Actually, does Δz_field for a center field give the interval between cell center or cell face?

I had thought about this, I think it would only make sense that it is the distance between cell faces, because the data lives on cell centers (and it should be centered / symmetric about cell centers). I should verify that, though. It returns `lg.∂x∂ξ.components.data.:9` for us, @simonbyrne, can you confirm that this is correct?

The MSE tables didn't need updated, so, if we have regression tests that use this, we should be in the clear.

Reply:
We do have regression test for this, this should be correct then.